MidasCivil_Adapter/Convert/ToBHoM/Properties/ToProfile.cs

```diff
@@ -69,10 +77,21 @@ public static IProfile ToProfile(List<string> sectionProfile, string shape, stri
                          System.Convert.ToDouble(sectionProfile[0]).LengthToSI(lengthUnit));
                     break;
                 case "H":
+                    double botFlangeWidth = System.Convert.ToDouble(sectionProfile[4]).LengthToSI(lengthUnit);
+                    double botFlangeThickness = System.Convert.ToDouble(sectionProfile[5]).LengthToSI(lengthUnit);
+                    if (botFlangeWidth < oM.Geometry.Tolerance.Distance)
+                    {
+                        botFlangeWidth = System.Convert.ToDouble(sectionProfile[1]).LengthToSI(lengthUnit);
+                    }
+                    if (botFlangeThickness < oM.Geometry.Tolerance.Distance)
+                    {
+                        botFlangeThickness = System.Convert.ToDouble(sectionProfile[3]).LengthToSI(lengthUnit);
+                    }
+
                     bhomProfile = Engine.Spatial.Create.FabricatedISectionProfile(
                         System.Convert.ToDouble(sectionProfile[0]).LengthToSI(lengthUnit), System.Convert.ToDouble(sectionProfile[1]).LengthToSI(lengthUnit),
-                        System.Convert.ToDouble(sectionProfile[4]).LengthToSI(lengthUnit), System.Convert.ToDouble(sectionProfile[2]).LengthToSI(lengthUnit),
-                        System.Convert.ToDouble(sectionProfile[3]).LengthToSI(lengthUnit), System.Convert.ToDouble(sectionProfile[5]).LengthToSI(lengthUnit), 0);
+                         botFlangeWidth, System.Convert.ToDouble(sectionProfile[2]).LengthToSI(lengthUnit),
+                        System.Convert.ToDouble(sectionProfile[3]).LengthToSI(lengthUnit), botFlangeThickness, 0);
```

Comment:
I think it would be better at the start to just do all the converts for the parameters, that way when you can sau
`botFlangeWidth = topFlangeWidth;`
Which is much clearer than some indexed `string`.